core/src/main/scala/spire/math/Real.scala

```diff
@@ -2,6 +2,8 @@ package spire
 package math
 
 import scala.math.{ScalaNumber, ScalaNumericConversions}
+import scala.collection.compat.immutable.LazyList
```

Comment:
I'm assuming this can be shortened to `scala.collection.immutable.LazyList` when we drop pre-2.13 support?

Reply:
it can be dropped entirely, in fact; `LazyList` (like `List` and `Vector`) has type and object aliases in the default scope